pkgs/development/python-modules/oterm/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub
+, poetry-core, textual, typer, python-dotenv, httpx
+, aiosql, aiosqlite, pyperclip, packaging
+, ruff, black, pytest, pytest-asyncio, textual-dev, aiohttp }:
+
+buildPythonPackage rec {
+  pname = "oterm";
+  version = "0.1.16";
+  src = fetchFromGitHub {
+    owner = "ggozad";
+    repo = "oterm";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-2jhsHX6o6I3nTDrFA7BG4D7J8Z5v2QuzKVBR0WTN2xA=";
+  };
+  propagatedBuildInputs = [
+    poetry-core
+    textual
+    typer
+    python-dotenv
+    httpx
+    aiosql
+    aiosqlite
+    pyperclip
+    packaging
+  ];
+  buildInputs = [
+    ruff
+    black
+    pytest
+    pytest-asyncio
+    textual-dev
+    aiohttp
+  ];
+  format = "pyproject";
+
+  meta = with lib; {
+    description = "A text-based terminal client for Ollama";
+    homepage = "https://github.com/ggozad/oterm";
```

```suggestion
    homepage = "https://github.com/ggozad/oterm";
    changelog = "https://github.com/ggozad/oterm/releases/tag/${version}";
```
Adding a changelog is helpful for future updates.